### ISSUE TITLE
Running test failed

### ISSUE THREAD

**Issue by cizixs** [coreos/go-iptables#31]
When I try to run test, the following error occurred: 

```
$ SUDO_PERMITTED=1 ./test
Building go-iptables...
Checking gofmt...
Running tests...
--- FAIL: TestChain (0.02s)
	iptables_test.go:107: testing iptables (hasWait=true, hasCheck=true)
	iptables_test.go:107: testing ip6tables (hasWait=true, hasCheck=true)
	iptables_test.go:107: testing iptables (hasWait=false, hasCheck=false)
	iptables_test.go:114: ListChains of Initial failed: exit status 4: Another app is currently holding the xtables lock. Perhaps you want to use the -w option?
--- FAIL: TestRules (0.02s)
	iptables_test.go:185: testing iptables (hasWait=true, hasCheck=true)
	iptables_test.go:185: testing ip6tables (hasWait=true, hasCheck=true)
	iptables_test.go:185: testing iptables (hasWait=false, hasCheck=false)
	iptables_test.go:205: ClearChain (of missing) failed: exit status 4: Another app is currently holding the xtables lock. Perhaps you want to use the -w option?
FAIL
coverage: 74.3% of statements
FAIL	github.com/coreos/go-iptables/iptables	0.050s
```

Running the failed command directly in terminal works well.

iptables version:

```
$ iptables --version
iptables v1.6.0
```

**Comment by gcmalloc** [comment]
linked to #24, which is a try to fix that.

**Comment by cizixs** [comment]
@gcmalloc Thanks, I'll keep following the PR.

**Comment by squeed** [comment]
Fixed in #38
